### DISCUSSION TITLE
how to check magento extension coding standard using docker

### DISCUSSION THREAD

**Discussion by bhushanmeetanshi** [markshust/docker-magento#1191] [Q&A] [answered]
how to check magento extension coding standard using docker.

for ex `src/app/code/VendorName/ExtensionName` is my directory then how can i execute command to check magento coding standard for vendor extension?

@markshust Please guide me.

**Comment by bhushanmeetanshi** [comment]
@YevhenZvieriev Please Guide me if u have any idea?

**Reply by YevhenZvieriev** [reply, marked as answer]
Hello, @bhushanmeetanshi 

I'm not sure it concerns Mark Shust's Docker Configuration for Magento, but I think this repo will be helpful for you :)

https://github.com/magento/magento-coding-standard

After installing pay attention to this block:

![image](https://github.com/user-attachments/assets/828f1c45-c738-4f51-983b-e9892158e5c2)
